README.md

```diff
@@ -908,9 +908,9 @@ ANSWERS.addComponent('FilterBox', {
   resetFilter: false,
   // Optional, the label to use for the reset button above, this will only display if searchOnChange is false
   resetFilterLabel: 'reset',
-  // Optional, show a reset-all button for the filter control, this will only display if searchOnChange is false
+  // Optional, show a reset-all button for the filter control. Defaults to displaying a reset button if searchOnChange is false.
```

Comment:
The commit message says the default behavior was to display a reset button if searchOnChange was true. Is that incorrect?

Reply:
Yeah this readme was wrong (for once I believe I am correct mwahaha!). 
1: in the config class config.resetFilters was being set to config.resetFacets, and defaulting to true. 
2: since the user should never be setting config.resetFacets, we can assume config.resetFilters was always true.
3: HOWEVER, in the template, the reset button would only render if showApplyButton is true.
4: showApplyButton is always set to !config.searchOnChange
5: Therefore the reset button only renders if !config.searchOnChange
6: Therefore the reset button would only render if searchOnChange is false

Reply:
Makes sense to me! I think we want to update the commit message for this item then. It says: "If resetFilters is not specified, it will default to its previous behavior, which was
displaying a reset button if searchOnChange was true"

Reply:
Oh shoot that's my bad

Reply:
updated!